src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const currentRent = 40;
+  const minDaysOff = 3;
+  const maxDaysOff = 7;
+  const minDiscountRent = 20;
+  const maxDiscountRent = 50;
```

Comment:
The names 'minDaysOff', 'maxDaysOff', 'minDiscountRent', and 'maxDiscountRent' are not very descriptive of their purpose. Consider renaming them to 'MIN_DAYS_FOR_DISCOUNT', 'MIN_DAYS_FOR_MAX_DISCOUNT', 'SMALL_DISCOUNT_AMOUNT', and 'LARGE_DISCOUNT_AMOUNT' respectively, to clearly indicate their role in the calculation.